src/components/sidebarHeader/SidebarHeader.tsx

```diff
@@ -0,0 +1,17 @@
+import searchIcon from "../../assets/search.svg";
+import "./SidebarHeader.css";
+import profilePic from "../../assets/profile-pic.webp";
+
+function SidebarHeader() {
+  return (
+    <nav className="navigation-container">
+      <div className="profile-section">
+        <img alt="profilePic" className="profile-picture" src={profilePic} />
```

Comment:
If we are adding a profile picture, name etc. We need to add login functionality and allow users to update them. Please remove this.